### PR TITLE
Admin can add users to a survey

### PR DESCRIPTION
**Pull request by nathanathan**

### CODE REVIEW

File: app/client/survey_admin/survey_admin_users.coffee [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+      console.log result
+    .fail (err)->
+      console.error(err)
+      toastr.error err.message
+    .always =>
+      @fetched.set(true)
+
+Template.survey_admin_users.helpers
+  fetched: ->
+    Template.instance().fetched.get()
```
Comment (jgoley) [inline app/client/survey_admin/survey_admin_users.coffee]: This helper shouldn't be necessary due to the globally registered helper here: 
https://github.com/ecohealthalliance/mobile-survey/blob/add-survey-users/app/client/helpers.coffee

**Comment by Broham** [comment]
@nathanathan when I try to add a user I get the following error in the console:
```
survey_user_edit.coffee:13 Uncaught TypeError: Cannot read property 'getSessionToken' of null
```
Are you getting this error?  I noticed that some surveys have users added to them so I have to assume it was working at some point.

**Comment by nathanathan** [comment]
@Broham I think somehow you got into a state where your meteor account is logged in while your parse account is logged out. I added some code that will log you out of your meteor account if your parse account is not logged in. After you log out then sign back in, this feature should work. Let me know if you run into more problems.

**Comment by Broham** [comment]
@nathanathan it looks good, but now there is a conflict.  If you resolve that I will go ahead and merge it

**Pull request merged**